cluster/images/conformance/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM BASEIMAGE
+FROM --platform=linux/ARCH BASEIMAGE
```

Comment:
IIRC this is not supported without buildkit, and I thought buildkit usage landed in a later release.

Reply:
https://github.com/kubernetes/kubernetes/blob/3c4329ca0ff19aa79a419065f06d5c947b1ed38b/cluster/images/conformance/Makefile#L59

release 1.19 is not build with buildkit, and it's not clear that the project would approve backporting buildkit as a requirement, because that will break other currently functioning release pipelines

Reply:
(er and to be clear and this image is not built with buildkit in the scripts, `docker build` is not typically buildkit unless you've opted to enable buildkit on the host, we do not depend on that even in HEAD, we call `docker buildx ...`)